tools/generators/add-codeowners.spec.ts

```diff
@@ -53,27 +53,27 @@ describe(`#addCodeowner`, () => {
     createCodeowners(tree);
 
     expect(tree.read(workspacePaths.github.codeowners, 'utf8')).toMatchInlineSnapshot(`
-      "/packages/react-one @org/team-one
-      /packages/react-one @org/team-two
+      "packages/react-one @org/team-one
+      packages/react-one @org/team-two
```

Comment:
Is it expected change?

Reply:
yes, there was bug within nx tree within testing environment now it's fixed/unified with "production"